lib/src/upgrade_alert.dart

```diff
@@ -26,6 +26,9 @@ class _UpgradeBase extends StatefulWidget {
   /// For debugging, display logging statements.
   final bool debugLogging;
 
+  /// Duration until alerting user again after later. Overrides `daysToAlertAgain`
+  final Duration durationToAlertAgain;
```

Comment:
This also needs to be added to classes UpgradeAlert and UpgradeCard.

Reply:
Thanks for catching! Done